apps/ecparam.c

```diff
@@ -207,6 +208,7 @@ int ecparam_main(int argc, char **argv)
     if (curve_name != NULL) {
         OSSL_PARAM params[4];
         OSSL_PARAM *p = params;
+        locale_t c_locale;
```

Comment:
@sil2100 points out, how come this is not declared static; and all other places have it as static.